### DISCUSSION TITLE
Lesson 0 Rinkeby Faucet Issues

### DISCUSSION THREAD

**Discussion by ghost** [smartcontractkit/full-blockchain-solidity-course-py#92] [Q&A] [answered]
Hi guys,

I ran into an odd issue with the Rinkeby Faucet funds.  I followed all of the instructions in Lesson 0 to get Rinkeby funds (post a tweet with my metamask address & the appropriate hashtags), and Rinkeby said it sent the funds, but they aren't appearing.  I tried to do it again, but Rinkeby says it won't send funds for 72 hours.  Not sure what could be causing it, any thoughts?

And yes, I double-checked to make sure I was on the Rinkeby network in my Metamask wallet.

Thanks,
Jarrett

**Comment by RobustCelery** [comment]
Try using the Kovan test network and use the Gitter faucet, this works for me [https://github.com/kovan-testnet/faucet](url)

**Reply by RobustCelery** [reply]
https://github.com/kovan-testnet/faucet

**Reply by ghost** [reply]
That worked for me, thanks!

**Comment by PatrickAlphaC** [comment, marked as answer]
Always go here for the most up to date faucets:
https://docs.chain.link/docs/link-token-contracts/#kovan
